content/news/posts/2025/autoupgrade-v7.0-release.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: "Update Assistant module v7.0.0 is available (Autoupgrade)"
+subtitle: "A New Era for PrestaShop updates"
+date:   2025-04-01
```

Comment:
I wanted to keep the yesterday's date @kpodemski, to avoid jokes about April fools :grimacing: